pkg/provision/kubernetes.go

```diff
@@ -0,0 +1,311 @@
+package provision
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/zos/pkg"
+	"github.com/threefoldtech/zos/pkg/stubs"
+)
+
+// Kubernetes reservation data
+type Kubernetes struct {
+	// Size of the vm, this defines the amount of vCpu, memory, and the disk size
```

Comment:
we should add the detail of the sizes here or a link to the document describing the sizes.